frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -76,8 +77,24 @@
         do-update-component #(do
                                (st/emit! (dwc/start-undo-transaction))
                                (st/emit! (dwl/update-component id))
-                               (st/emit! (dwl/sync-file current-file-id))
+                               (st/emit! (dwl/sync-file current-file-id
+                                                        (:component-file shape)))
                                (st/emit! (dwc/commit-undo-transaction)))
+        confirm-update-remote-component #(do
+                                           (st/emit! (dwl/update-component id))
+                                           (st/emit! (dwl/sync-file current-file-id
+                                                                    (:component-file shape)))
+                                           (st/emit! (dwl/sync-file (:component-file shape)
+                                                                    (:component-file shape))))
```

Comment:
Can be defined using `emitf`:
```clojure
      confirm-update-remote-component
      (st/emitf (dwl/update-component id)
                (dwl/sync-file current-file-id (:component-file shape))
                (dwl/sync-file (:component-file shape) (:component-file shape)))
```

just a **cosmetic change**

Reply:
Done. Also changed other `#(st/emit!`  by `(st/emitf`.